src/extension.ts

```diff
@@ -60,7 +63,7 @@ export async function activate(context: vscode.ExtensionContext) {
         Logger.initLogger(loggerConfig);
 
         log = Logger.getAppLogger();
-        log.debug("Initialized logger from VSCode extension");
+        log.debug(localize("log.debug", "Initialized logger from VSCode extension"));
```

Comment:
This should be re-named  to something more specific even though it's at top level outside of a function, for example: "initialization.log.debug"